processing/spikesSiliconProbesV2.m

```diff
@@ -0,0 +1,20 @@
+function [channels, artifacts] = spikesSiliconProbesV2(sourceFile, spikesFile)
+% Spike detection callback for silicon probes.
+% AE 2011-10-26
+
+
+channels = 1 : 32;
+
+% Extract the common reference - mean across channels at each timepoint
+extract_common_referenceSP(sourceFile);
```

Comment:
also i think you should be able to pass your fixes back to extract_common_reference.m. the only difference between calling it for utah arrays versus these depth probes are the set of channels, which you can pass into the method.